fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/CascadesCostModel.java

```diff
@@ -223,6 +236,47 @@ private OptionalInt comparePrimaryScanToIndexScan(@Nonnull Map<Class<? extends R
         return OptionalInt.empty();
     }
 
+    private OptionalInt compareInUnion(@Nonnull final RelationalExpression leftExpression,
+                                       @Nonnull final RelationalExpression rightExpression) {
+        if (!(leftExpression instanceof RecordQueryInUnionPlan)) {
+            return OptionalInt.empty();
+        }
+
+        //
+        // If both an InUnions we just return 0
```

Comment:
Which means what? That both are kept in consideration?

Reply:
Yes -- updated the comment.